core/src/main/java/io/stargate/core/CoreActivator.java

```diff
@@ -0,0 +1,28 @@
+package io.stargate.core;
+
+import io.stargate.core.metrics.api.Metrics;
+import io.stargate.core.metrics.impl.MetricsImpl;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceEvent;
+import org.osgi.framework.ServiceListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CoreActivator implements BundleActivator, ServiceListener {
+  private static final Logger logger = LoggerFactory.getLogger(CoreActivator.class);
+
+  @Override
+  public void start(BundleContext context) {
```

Comment:
what is a long term plan for this module? Would it have only a metrics module? If yes, then I think it should be called just MetricsActivator.

Reply:
afaik, there can be one activator per module, so all core services will be registered here. For now, only the metrics...